app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1> Mechanic Info </h1>
+
+<p>Name: <%= @mechanic.name %></p>
+<p>Years of Experience: <%= @mechanic.years_experience %></p>
+
+<p>Assigned Rides</p>
+  <% @mechanic.current_rides.each do |ride| %>
```

Comment:
hmmm, could have just done `@mechanic.rides.each do....` since `.rides` is the association you established.